guides/common/modules/con_migrating-from-internal-databases-to-external-databases.adoc

```diff
@@ -21,7 +21,7 @@ endif::[]
 To migrate from the default internal databases to external databases, you must complete the following procedures:
 
 . xref:preparing-a-host-for-external-databases_{context}[].
-Prepare a {RHEL} 7 server to host the external databases.
+Prepare a {EL} 8 or {EL} 7 server to host the external databases.
```

Comment:
Please hide this change for orcharhino. I am unsure about other build targets.

Reply:
See https://github.com/theforeman/foreman-documentation/milestone/2 for that overview. I just changed it to a table, but I don't know the values for Orcharhino. Please check them. I assume you have the privileges to edit the milestone, but if you don't, please let me know.

Reply:
Addressed for this change :+1: